### ISSUE TITLE
Training goes to 0 and accuracy to 100 soon after first batch

### ISSUE THREAD

**Issue by codeprb** [pytorch/examples#768]
I am training a Resnet model with Imagenet dataset. For some reason the accuracy is 100 after the first batch and loss go to zero forever. here are the details. Any help is greatly appreciated. What could be causing this error. 

2020-04-30 15:57:05 - DEBUG - run arguments: Namespace(batch_size=256, dataset='imagenet', epochs=1, evaluate=None, gpus='0', input_size=None, lr=0.1, model='resnet_binary', model_config='', momentum=0.9, optimizer='SGD', print_freq=10, results_dir='./results', resume='', save='2020-04-30_15-57-05', start_epoch=0, type='torch.FloatTensor', weight_decay=0.0001, workers=8)
2020-04-30 15:57:05 - INFO - creating model resnet_binary
2020-04-30 15:57:05 - INFO - created model with configuration: {'input_size': None, 'dataset': 'imagenet'}
2020-04-30 15:57:05 - INFO - number of parameters: 11689512
2020-04-30 15:57:05 - INFO - training regime: {0: {'optimizer': 'SGD', 'lr': 0.1, 'weight_decay': 0.0001, 'momentum': 0.9}, 30: {'lr': 0.01}, 60: {'lr': 0.001, 'weight_decay': 0}, 90: {'lr': 0.0001}}
2020-04-30 15:57:05 - DEBUG - OPTIMIZER - setting method = SGD
2020-04-30 15:57:05 - DEBUG - OPTIMIZER - setting lr = 0.1
2020-04-30 15:57:05 - DEBUG - OPTIMIZER - setting momentum = 0.9
2020-04-30 15:57:05 - DEBUG - OPTIMIZER - setting weight_decay = 0.0001
2020-04-30 15:58:06 - INFO - TRAINING - Epoch: [0][0/176]	Time 60.928 (60.928)	Data 2.290 (2.290)	Loss 71.5708 (71.5708)	Prec@1 0.391 (0.391)	Prec@5 0.391 (0.391)
2020-04-30 16:07:37 - INFO - TRAINING - Epoch: [0][10/176]	Time 54.159 (57.415)	Data 0.007 (0.211)	Loss 0.0000 (6.5064)	Prec@1 100.000 (90.945)	Prec@5 100.000 (90.945)
2020-04-30 16:16:51 - INFO - TRAINING - Epoch: [0][20/176]	Time 57.379 (56.445)	Data 0.000 (0.111)	Loss 0.0000 (3.4081)	Prec@1 100.000 (95.257)	Prec@5 100.000 (95.257)
2020-04-30 16:26:52 - INFO - TRAINING - Epoch: [0][30/176]	Time 59.362 (57.632)	Data 0.002 (0.076)	Loss 0.0000 (2.3087)	Prec@1 100.000 (96.787)	Prec@5 100.000 (96.787)
2020-04-30 16:35:55 - INFO - TRAINING - Epoch: [0][40/176]	Time 53.450 (56.818)	Data 0.002 (0.058)	Loss 0.0000 (1.7456)	Prec@1 100.000 (97.571)	Prec@5 100.000 (97.571)
2020-04-30 16:44:54 - INFO - TRAINING - Epoch: [0][50/176]	Time 54.136 (56.241)	Data 0.000 (0.047)	Loss 0.0000 (1.4033)	Prec@1 100.000 (98.047)	Prec@5 100.000 (98.047)
2020-04-30 16:53:55 - INFO - TRAINING - Epoch: [0][60/176]	Time 54.255 (55.887)	Data 0.002 (0.039)	Loss 0.0000 (1.1733)	Prec@1 100.000 (98.367)	Prec@5 100.000 (98.367)
2020-04-30 17:02:56 - INFO - TRAINING - Epoch: [0][70/176]	Time 54.364 (55.641)	Data 0.000 (0.034)	Loss 0.0000 (1.0080)	Prec@1 100.000 (98.597)	Prec@5 100.000 (98.597)
2020-04-30 17:11:56 - INFO - TRAINING - Epoch: [0][80/176]	Time 53.740 (55.440)	Data 0.003 (0.030)	Loss 0.0000 (0.8836)	Prec@1 100.000 (98.770)	Prec@5 100.000 (98.770)
2020-04-30 17:20:58 - INFO - TRAINING - Epoch: [0][90/176]	Time 54.247 (55.305)	Data 0.000 (0.027)	Loss 0.0000 (0.7865)	Prec@1 100.000 (98.905)	Prec@5 100.000 (98.905)
2020-04-30 17:30:00 - INFO - TRAINING - Epoch: [0][100/176]	Time 54.309 (55.198)	Data 0.000 (0.024)	Loss 0.0000 (0.7086)	Prec@1 100.000 (99.014)	Prec@5 100.000 (99.014)
2020-04-30 17:39:02 - INFO - TRAINING - Epoch: [0][110/176]	Time 54.098 (55.102)	Data 0.000 (0.022)	Loss 0.0000 (0.6448)	Prec@1 100.000 (99.103)	Prec@5 100.000 (99.103)
2020-04-30 17:48:01 - INFO - TRAINING - Epoch: [0][120/176]	Time 53.637 (55.008)	Data 0.000 (0.021)	Loss 0.0000 (0.5915)	Prec@1 100.000 (99.177)	Prec@5 100.000 (99.177)
2020-04-30 17:57:01 - INFO - TRAINING - Epoch: [0][130/176]	Time 53.865 (54.926)	Data 0.000 (0.019)	Loss 0.0000 (0.5463)	Prec@1 100.000 (99.240)	Prec@5 100.000 (99.240)
2020-04-30 18:06:02 - INFO - TRAINING - Epoch: [0][140/176]	Time 55.712 (54.871)	Data 0.000 (0.018)	Loss 0.0000 (0.5076)	Prec@1 100.000 (99.294)	Prec@5 100.000 (99.294)
2020-04-30 18:18:48 - INFO - TRAINING - Epoch: [0][150/176]	Time 111.563 (56.310)	Data 0.002 (0.017)	Loss 0.0000 (0.4740)	Prec@1 100.000 (99.340)	Prec@5 100.000 (99.340)
2020-04-30 18:37:27 - INFO - TRAINING - Epoch: [0][160/176]	Time 112.165 (59.761)	Data 0.001 (0.016)	Loss 0.0000 (0.4445)	Prec@1 100.000 (99.381)	Prec@5 100.000 (99.381)
2020-04-30 20:48:58 - INFO - TRAINING - Epoch: [0][170/176]	Time 52.576 (102.412)	Data 0.003 (0.015)	Loss 0.0000 (0.4185)	Prec@1 100.000 (99.417)	Prec@5 100.000 (99.417)
2020-04-30 20:53:35 - INFO - EVALUATING - Epoch: [0][0/20]	Time 25.402 (25.402)	Data 2.827 (2.827)	Loss 0.0000 (0.0000)	Prec@1 100.000 (100.000)	Prec@5 100.000 (100.000)
2020-04-30 20:57:27 - INFO - EVALUATING - Epoch: [0][10/20]	Time 21.789 (23.421)	Data 0.012 (0.261)	Loss 0.0000 (0.0000)	Prec@1 100.000 (100.000)	Prec@5 100.000 (100.000)
2020-04-30 21:00:42 - INFO - 
 Epoch: 1	Training Loss 0.4072 	Training Prec@1 99.433 	Training Prec@5 99.433 	Validation Loss 0.0000 	Validation Prec@1 100.000 	Validation Prec@5 100.000

**Comment by Little-thinker** [comment]
> 我正在使用Imagenet数据集训练Resnet模型。由于某些原因，第一批后的精度为100，并且损失永远为零。这是详细信息。任何帮助是极大的赞赏。是什么导致此错误。
> 
> 2020-04-30 15:57:05-调试-运行参数：命名空间（batch_size = 256，dataset ='imagenet'，epochs = 1，valuate = None，gpus ='0'，input_size = None，lr = 0.1， model ='resnet_binary'，model_config =''，动量= 0.9，优化器='SGD'，print_freq = 10，results_dir ='。/ results'，resume =''，save ='2020-04-30_15-57-05 '，start_epoch = 0，type ='torch.FloatTensor'，weight_decay = 0.0001，worker = 8）
> 2020-04-30 15:57:05-信息-创建模型resnet_binary
> 2020-04-30 15:57:05-信息-使用配置创建的模型：{'input_size'：无，'dataset'：'imagenet'}
> 2020-04-30 15:57:05-INFO-参数数量：11689512
> 2020-04-30 15:57:05-信息-培训制度：{0：{'optimizer'：'SGD'，'lr'：0.1，'weight_decay'：0.0001，'momentum'：0.9}，30：{ 'lr'：0.01}，60：{'lr'：0.001，'weight_decay'：0}，90：{'lr'：0.0001}}
> 2020-04-30 15 : 57:05-调试-优化-设置方法= SGD
> 2020-04-30 15:57:05-调试-优化-设置lr = 0.1
> 2020-04-30 15:57:05-调试-优化-设置动量= 0.9
> 2020-04-30 15:57:05 -调试-优化-设置weight_decay = 0.0001
> 2020-04-30 15:58:06-信息-培训-时期：[0] [0/176]时间60.928（60.928）数据2.290（2.290）损失71.5708（71.5708）精确@ 1 0.391（0.391）Prec @ 5 0.391（0.391）
> 2020-04-30 16:07:37-信息-培训-纪元：[0] [10/176]时间54.159（57.415）数据0.007（0.211）损失0.0000（6.5064）Prec @ 1 100.000（90.945）Prec @ 5 100.000（90.945）
> 2020-04-30 16:16:51-信息-训练-时期：[0] [20/176]时间57.379（56.445）数据0.000（0.111）损失0.0000（3.4081）Prec @ 1 100.000（95.257 ）Prec @ 5 100.000（95.257）
> 2020-04-30 16:26:52-信息-培训-时期：[0] [30/176]时间59.362（57.632）数据0.002（0.076）损失0.0000（2.3087）Prec @ 1 100.000（96.787）Prec @ 5 100.000（96.787）
> 2020-04-30 16:35:55-信息-培训-时期：[0] [40/176]时间53.450（56.818）数据0.002（0.058）损失0.0000（ 1.7456）Prec @ 1 100.000（97.571）Prec @ 5 100.000（97.571）
> 2020-04-30 16:44:54-信息-培训-纪元：[0] [50/176]时间54.136（56.241）数据0.000（0.047）损失0.0000（1.4033）Prec @ 1 100.000（98.047）Prec @ 5 100.000（98.047）
> 2020-04-30 16:53:55-信息-培训-纪元：[0] [60/176]时间54.255（55.887）数据0.002（0.039）损失0.0000（1.1733）Prec @ 1 100.000（98.367） ）Prec @ 5 100.000（98.367）
> 2020-04-30 17:02:56-信息-培训-纪元：[0] [70/176]时间54.364（55.641）数据0.000（0.034）损失0.0000（1.0080）Prec @ 1 100.000（98.597）Prec @ 5 100.000（98.597）
> 2020-04-30 17:11:56-信息-培训-时期：[0] [80/176]时间53.740（55.440）时间数据0.003（0.030）损失0.0000（ 0.8836）Prec @ 1 100.000（98.770）Prec @ 5 100.000（98.770）
> 2020-04-30 17:20:58-信息-培训-纪元：[0] [90/176]时间54.247（55.305）数据0.000（0.027）损失0.0000（0.7865）Prec @ 1 100.000（98.905）Prec @ 5 100.000（98.905）
> 2020-04-30 17:30:00-信息-培训-纪元：[0] [100/176]时间54.309（55.198）数据0.000（0.024）损失0.0000（0.7086）Prec @ 1 100.000（99.014 ）Prec @ 5 100.000（99.014）
> 2020-04-30 17:39:02-信息-培训-时期：[0] [110/176]时间54.098（55.102）数据0.000（0.022）损失0.0000（0.6448）Prec @ 1 100.000（99.103）Prec @ 5 100.000（99.103）
> 2020-04-30 17:48:01-信息-训练-时期：[0] [120/176]时间53.637（55.008）数据0.000（0.021）损失0.0000（ 0.5915）Prec @ 1 100.000（99.177）Prec @ 5 100.000（99.177）
> 2020-04-30 17:57:01-信息-培训-纪元：[0] [130/176]时间53.865（54.926）数据0.000（0.019）损失0.0000（0.5463）Prec @ 1 100.000（99.240）Prec @ 5 100.000（99.240）
> 2020-04-30 18:06:02-信息-培训-时期：[0] [140/176]时间55.712（54.871）数据0.000（0.018）损失0.0000（0.5076）Prec @ 1 100.000（99.294 ）Prec @ 5 100.000（99.294）
> 2020-04-30 18:18:48-信息-培训-纪元：[0] [150/176]时间111.563（56.310）数据0.002（0.017）损失0.0000（0.4740）Prec @ 1 100.000（99.340）Prec @ 5 100.000（99.340）
> 2020-04-30 18:37:27-信息-训练-时期：[0] [160/176]时间112.165（59.761）数据0.001（0.016）损失0.0000（ 0.4445）Prec @ 1 100.000（99.381）Prec @ 5 100.000（99.381）
> 2020-04-30 20:48:58-信息-培训-纪元：[0] [170/176]时间52.576（102.412）数据0.003（0.015）损失0.0000（0.4185）Prec @ 1 100.000（99.417）Prec @ 5 100.000（99.417）
> 2020-04-30 20:53:35-信息-评估-时期：[0] [0/20]时间25.402（25.402）数据2.827（2.827）损失0.0000（0.0000）Prec @ 1 100.000（100.000 ）Prec @ 5 100.000（100.000）
> 2020-04-30 20:57:27-信息-评估-时期：[0] [10/20]时间21.789（23.421）数据0.012（0.261）损失0.0000（0.0000）Prec @ 1 100.000（100.000）Prec @ 5 100.000（100.000）
> 2020-04-30 21
> : 00:42-INFO-纪元：1培训损失0.4072培训Prec @ 1 99.433培训Prec @ 5 99.433验证损失0.0000验证Prec @ 1 100.000验证精确@ 5 100.000

Hello ! Has the problem of training accuracy you mentioned been resolved?

**Comment by suraj813** [comment]
Closing due to lack of activity, feel free to reopen if necessary.
